### ISSUE TITLE
Monthly Recurrence Deletes all JSON Events (JSON Build)

### ISSUE THREAD

**Issue by rico161** [waldronmatt/FullCalendar-BS4-PHP-MySQL-JSON#16]
I noticed in the index-json.php file the monthly recurrence option was commented out. I uncommented it because this is a feature i actually wanted, but when i tried to set a monthly event it deleted all the other events. I'm guessing this is why it was commented out, but is there any way to fix this? It would actually be a very useful feature, much more than the weekly recurrence.

If it doesn't work in the JSON build, would it by chance work in the SQL build? I tried the SQL version first but even though i set up all the database connections correctly it would redirect to a blank page upon event creation, so we went with the JSON build instead.

**Comment by waldronmatt** [comment]
You are correct. I had it commented out because it is not operational. I paused work on this because there are better methods to implement recurrences. You might want to have a look at FullCalendar's RRule https://fullcalendar.io/docs/rrule-plugin. Neither does this work on the MySQL version unfortunately.
